### ISSUE TITLE
Is the project still active?

### ISSUE THREAD

**Issue by inspirit** [taogashi/winter-offensive#1]
I was wondering if u still maintain the project and planning on further updates?
i had some ideas on SemiDense tracking

**Comment by taogashi** [comment]
Hi inspirit, this repo is a modified version of [SVO](https://github.com/uzh-rpg/rpg_svo), I expect to incorporate edges to get a better tracking performance, but oops it turns out to be degraded, another repo referred is [LSD-SLAM](https://github.com/tum-vision/lsd_slam). Any good idea?

**Comment by inspirit** [comment]
yeah i tried your changes but Tracking itself really got worse with edges. 
you can get a bit better results using LSD Line detector instead of simple gradient-magnitudes. still tracking is not reliable :( 
but the main weak point is Pure Rotation motion in most of SLAM systems and SVO easily lose tracking in such cases no matter how much points you have in the previous frames

**Comment by taogashi** [comment]
Yes you're right. But I thought LSD line detector is somehow the same as simple gradient-magnitudes. My purpose is to extend the scenario that SVO could work in,  such as those contain rich edges but few corners.

**Comment by inspirit** [comment]
Well LSD is better since it can robustly detect stable Line segments, which u can later split into multiple Edgelets. it worked for me much better than just gradients.

**Comment by inspirit** [comment]
I also noticed that Edgelet tracking/matching (1D alignment) in general very poorly converge. just for testing tried to use edges instead of corners as general features and it fails after 3-5 seconds

**Comment by taogashi** [comment]
you mean only using edgelets to track the motion? SVO framework doesn't work with edgelets

**Comment by inspirit** [comment]
they have EDGELET feature type. and they treat it with 1D alignment method instead of 2D alignment that they use for corners.

**Comment by taogashi** [comment]
1D alignment is not as accuracy as 2D, and 3D position of edgelet feature tends to diverge in structure-only BA in SVO's tracking stage, I think.

**Comment by inspirit** [comment]
i've tried to forward EDGELET features from DepthMapManager to MapPointCandidates the same way original DepthFilter is doing and it seems to help to cope with situations where not enough corners can be detected/tracked but we have edges there.
the main problem with SemiDenseTracking is poor features distributions across viewport i guess, thats why it usually doesnt help in tracking
